src/router/index.js

```diff
@@ -25,9 +25,9 @@ const router = createRouter({
       component: Dashboards,
     },
     {
-      path: '/dashboards/dashboard',
-      name: 'dashboard',
+      path: '/dashboards/dashboard/:id',
       component: Dashboard,
+      props: true,
```

Comment:
Instead of `true`, it is better for readability and security if props are specified here.